app/UiTPAS/UiTPASServiceProvider.php

```diff
@@ -72,8 +73,8 @@ public function register(): void
             );
         });
 
-        $this->app->singleton(ActivateUiTPASClientHandler::class, function () {
-            return new ActivateUiTPASClientHandler(
+        $this->app->singleton(ActivateUdbOrganizer::class, function () {
+            return new ActivateUdbOrganizer(
```

Comment:
I don't think we need to register Nova actions?

Reply:
In this case we do to resolve ` ClientCredentialsContextFactory::getUitIdProdContext()`, there is also a test context.